pkg/commands/compute/init.go

```diff
@@ -68,14 +70,22 @@ func NewInitCommand(parent argparser.Registerer, g *global.Data) *InitCommand {
 // Exec implements the command interface.
 func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	var introContext string
+	var isExistingService bool
```

Comment:
NIT: maybe combine into single `var`...

```suggestion
	var (
		introContext string
		isExistingService bool
	)
```